### PR TITLE
Read and Print all Data

### PR DESCRIPTION
**Pull request by Greenfin17**
## Description
- Implemented authentication
- Created navbar
- Created modules for each view
- Created cards for each view ( dinosaurs, vendors, equipment, staff, rides )
- Created json data for each view
- Add functionality to print cards to the DOM
- Created event listener for the navbar buttons
- Created events folder with domEvents and navEvents
- Created functions to retrieve data from Firebase with promises
- Initial basic styling for consistency
- Added pull request template
- Added issue ticket template

## Related Issue
- https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/milestone/4
- https://github.com/nss-evening-cohort-14/jurrasic-world-dashboard-jurrasic-world-dashboard/milestone/3

## Motivation and Context
- Sets initial framework for app and data organization.

## How Can This Be Tested?
- https://jurassic-world-dashboard.netlify.app/#
- git fetch origin development, npm start

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,10 +4,9 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="shortcut icon" href="#">
-  <title>Test</title>
+  <title>Jurassic World Dashboard</title>
```
Comment (drteresavasquez) [inline src/index.html:7]: Nice catch! 👍🏼

File: src/javascripts/components/cards/dinos.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const showDinos = (array) => {
+  // document.querySelector('#add-button').innerHTML = '<button class="btn btn-success btn-lg mb-4" id="add-board-btn">Add Dinosaur</button>';
```
Comment (drteresavasquez) [inline src/javascripts/components/cards/dinos.js]: Even if you plan to use this later, let's remove all commented out code. It does not belong on the production branch

File: src/javascripts/components/cards/dinos.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  document.querySelector('#card-container').innerHTML = '';
+  // document.querySelector('#form-container').innerHTML = '';
```
Comment (drteresavasquez) [inline src/javascripts/components/cards/dinos.js]: Same here for commented code

File: src/javascripts/helpers/data/vendorData.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+// vendorData.js
```
Comment (drteresavasquez) [inline src/javascripts/helpers/data/vendorData.js]: Remove and get rid of the extra lines

File: src/javascripts/main.js [hunk 5]
```diff
@@ -1,15 +1,14 @@
 // USE WITH FIREBASE AUTH
 // import checkLoginStatus from './helpers/auth';
 import 'bootstrap'; // import bootstrap elements and js
-
 import '../styles/main.scss';
+import checkLoginStatus from './helpers/auth';
```
Comment (drteresavasquez) [inline src/javascripts/main.js:3]: You have this commented out on line 2. Delete one of them. Also, is the app starting here or checking the login status?

File: src/javascripts/helpers/auth.js [hunk 6]
```diff
@@ -3,15 +3,18 @@ import 'firebase/auth';
...
 import logoutButton from '../components/logoutButton';
 import firebaseConfig from './apiKeys';
+import startApp from './views/startApp';
 
 const checkLoginStatus = () => {
   firebase.initializeApp(firebaseConfig);
   firebase.auth().onAuthStateChanged((user) => {
     if (user) {
       // person is logged in do something...
+      startApp();
```
Comment (drteresavasquez) [inline src/javascripts/helpers/auth.js:13]: You're calling this is various places. Make sure that you either call it here on in the main.js

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
Few changes

**Comment by drteresavasquez** [comment]
I am letting you get away with the buttons showing on the unauthed site, but you may want to take care of those not showing until a user is authenticated sooner rather than later.

**Comment by Greenfin17** [comment]
Issues resolved. Requesting approval.

**Review by drteresavasquez — APPROVED** [review]

**Pull request merged**